plugins/ZynAddSubFx/ZynAddSubFx.cpp

```diff
@@ -142,6 +142,11 @@ ZynAddSubFxInstrument::ZynAddSubFxInstrument(
 
 	connect( instrumentTrack()->pitchRangeModel(), SIGNAL( dataChanged() ),
 			this, SLOT( updatePitchRange() ), Qt::DirectConnection );
+
+	// ZynAddSubFX's internal value that LMMS's FREQ knob controls
+	// isn't set properly when the patch is first loaded in,
```

Comment:
"the patch" - not clear what was meant here, need better description

Reply:
What about preset?

Reply:
LMMS refers to patches as "instruments" (Clear instrument, Instrument bank, etc), you might wanna go with that.